indico/modules/rb_new/client/js/common/bookings/actions.js

```diff
@@ -44,6 +45,10 @@ export const UPDATE_BOOKING_SUCCESS = 'bookings/UPDATE_BOOKING_SUCCESS';
 export const UPDATE_BOOKING_ERROR = 'bookings/UPDATE_BOOKING_ERROR';
 export const UPDATED_BOOKING_RECEIVED = 'bookings/UPDATED_BOOKING_RECEIVED';
 
+export const BOOKING_OCCURRENCE_STATE_CHANGE_REQUEST = 'bookings/BOOKING_OCCURRENCE_STATE_CHANGE_REQUEST';
+export const BOOKING_OCCURRENCE_STATE_CHANGE_SUCCESS = 'bookings/BOOKING_OCCURRENCE_STATE_CHANGE_SUCCESS';
+export const BOOKING_OCCURRENCE_STATE_CHANGE_ERROR = 'bookings/BOOKING_OCCURRENCE_STATE_CHANGE_ERROR';
+export const BOOKING_OCCURRENCE_STATE_UPDATED = 'bookings/BOOKING_OCCURRENCE_STATE_UPDATED';
```

Comment:
Shouldn't this action be handled in the calendar-related reducers as well?
E.g. when you are in the calendar and cancel an occurrence. Then this occurrence should disappear from the calendar (and ideally change styles according to the new state if inactive occurrences are enabled)

Reply:
Separate task: #3753